packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -225,38 +239,64 @@ class _OverlayEntryWidget extends StatefulWidget {
   const _OverlayEntryWidget({
     required Key key,
     required this.entry,
+    required this.overlayState,
     this.tickerEnabled = true,
   }) : assert(key != null),
        assert(entry != null),
        assert(tickerEnabled != null),
        super(key: key);
 
   final OverlayEntry entry;
+  final OverlayState overlayState;
   final bool tickerEnabled;
 
   @override
   _OverlayEntryWidgetState createState() => _OverlayEntryWidgetState();
 }
 
 class _OverlayEntryWidgetState extends State<_OverlayEntryWidget> {
+  late _RenderTheatre _theatre;
```

Comment:
I think this should be Theater in order to use the most common US spelling https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#spell-words-in-identifiers-and-comments-correctly